test/test_profiler.py

```diff
@@ -99,6 +102,98 @@ def forward(self, x):
 
         torch._C._set_graph_executor_optimize(prev_opt)
 
+    def test_python(self):
```

Comment:
please choose a more descriptive name

Reply:
@ilia-cher Thanks! Renamed to "test_high_level_trace".